src/components/CellWithPopover/CellWithPopover.scss

```diff
@@ -1,8 +1,12 @@
 .ydb-cell-with-popover {
-    display: flex;
+    display: inline-flex;
 
     max-width: 100%;
 
+    &_full-width {
+        display: flex;
+    }
+
```

Comment:
We need `flex` to display progress bars correctly, for other values we need `inline-flex` for proper align inside table